### PR TITLE
Fix volumeMounts format in helm chart

### PR DESCRIPTION
**Pull request by wuminorb**
### What changes are proposed in this pull request?
fix volumeMounts format in helm chart

### Why are the changes needed?
fix https://github.com/Alluxio/alluxio/issues/14682

### Does this PR introduce any user facing changes?
Enable metrics in helm chart can working

### CODE REVIEW

**Comment by wuminorb** [comment]
alluxio-bot, check this please

**Review by jiacheliu3 — CHANGES_REQUESTED** [review]
The change looks good to me. Could you also bump up the version in https://github.com/Alluxio/alluxio/blob/master/integration/kubernetes/helm-chart/alluxio/CHANGELOG.md and https://github.com/Alluxio/alluxio/blob/master/integration/kubernetes/helm-chart/alluxio/Chart.yaml

Thanks for the improvements!

**Review by ZhuTopher — CHANGES_REQUESTED** [review]
+1 with Jiacheng's request to bump the version in `Chart.yaml` and update the `CHANGELOG.md`. Make sure you pull the latest version before doing so as well!

File: integration/kubernetes/helm-chart/alluxio/CHANGELOG.md [hunk 1]
```diff
@@ -238,3 +238,6 @@
 
 - Remove usage of Helm hook annotations in Charts
 
+0.6.36
+
+- Fix volumeMounts format in master statefulset
```
Comment (jiacheliu3) [inline integration/kubernetes/helm-chart/alluxio/CHANGELOG.md:243]:
```suggestion
- Fix volumeMounts indentations in master statefulset
```

**Review by jiacheliu3 — APPROVED** [review]
LGTM with a minor comment. Just commit that and it's good for merging.

Thanks for the fix!

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Pull request merged**